Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
+        --build.exclude_dir "node_modules" \
+        --build.include_ext "go" \
+        --misc.clean_on_exit true
+    requires:
+      vars: [GITHUB_TOKEN, OPENAI_API_KEY]
+  test:
+    cmds:
+      - go test -v ./...
+
+  i18n-extract:
+    cmds:
+      - goi18n extract -outdir internal/localization -sourceLanguage en -format toml
```

Comment:
In the `i18n-extract`, `i18n-merge`, and `i18n-merge-finish` commands, make sure you handle the case where there are no `.toml` files to prevent errors during execution. You may want to include conditional checks or error handling to inform the user if there are no files to process.